src/ucp/rndv/rndv.c

```diff
@@ -266,6 +266,13 @@ ucp_rndv_adjust_zcopy_length(size_t min_zcopy, size_t max_zcopy, size_t align,
     return result_length;
 }
 
+static void ucp_rndv_ack_comp_cb(uct_completion_t *comp)
```

Comment:
maybe rename it to ucp_proto_comp_cb, and call ack_req->send.proto.comp_cb?

Reply:
ok, done